routers/api/v1/user/repo.go

```diff
@@ -66,6 +70,20 @@ func ListMyRepos(ctx *context.APIContext) {
 	listUserRepos(ctx, ctx.User)
 }
 
+// ListOrgRepos - list the repositories of an organization.
+func ListOrgRepos(ctx *context.APIContext) {
+	// swagger:route GET /orgs/{org}/repos orgListRepos
+	//
+	//     Produces:
+	//     - application/json
+	//
+	//     Responses:
+	//       200: RepositoryList
+	//       500: error
+
+	listUserRepos(ctx, ctx.Org.Organization)
```

Comment:
I have made a PR #2117 to enforce that ctx.Org.Organization is an org. So this detail could be ignore and the check up under /users/{username}/repo could be make in another PR.